### PR TITLE
[plugin-web-app] Add Edge (Chromium) support

### LINKED ISSUE #372: Edge (Chronium) support required for Vividus (0.2.1)

### PR DESCRIPTION
**Pull request by Ulyana-Yeryksonava**
Closes #372

### CODE REVIEW

File: vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/edge/chromium/profile.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+selenium.browser=edge_chromium
+
+selenium.grid.capabilities.version=13
+selenium.grid.capabilities.browserName=MicrosoftEdge
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/edge/chromium/profile.properties]: is it required to be set?

File: vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/edge/chromium/profile.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+selenium.browser=edge_chromium
+
+selenium.grid.capabilities.platformName=ANY
```
Comment (ikalinin1) [inline vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/edge/chromium/profile.properties]: Why actually we need platfromName and version of the browser?

Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/edge/chromium/profile.properties]: Also please consider addition of screenshot shooting strategy

Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/edge/chromium/profile.properties]: For chrome we use latest version for example

**Pull request merged**